en/contributing-writing.md

```diff
@@ -71,39 +79,43 @@ Text goes here blah blah blah
 - The table of contents will contain only level 2 headers.
 - The metadata section is terminated by a blank line.
 
-### Sections
 
-Text is organised into sections. These are auto-generated, there is nothing
-extra you need to do:
+## Headers
+
+Headers are simple to create:
 
     # Top level header (typically the same as the Title element)
     ## Second level header
     ### Third level header
 
-### Code blocks
 
-Code blocks are created using the code-fencing markup of three backticks,
-followed by the type of code:
+## Code blocks
+
+A code block is enclosed by three backticks and includes the *type* of code:
 
     ```bash
     maas command do something
     maas command do something else
     ```
 
-The most common *types* used are: `bash`, `yaml`, `json`, and `no-highlight`.
+The most common types used are: `bash`, `yaml`, `json`, and `no-highlight`.
+The last is like a miscellaneous type. It is often used to display command
+output.
 
-### Inline code
+
+## Inline code
 
 Use a backtick to `inline filenames and other literals` like this:
 
 ```no-highlight
 Use a backtick to `inline filenames and other literals`.
 ```
 
-### Admonishments
 
-Admonishments are used to distinguish information from the rest of the text.
-They use the following format:
+## Admonishments
```

Comment:
For some reason, the code that surround these, and DOCBOOK, refer to these as admonitions. I have no clue why. But it may be worth adding a "(or 'admonition')" at some point